### PR TITLE
Make datachain logo show up properly on Safari in dark mode

### PR DESCRIPTION
**Pull request by djsauble**
Dark mode:
image

Light mode:
image

### CODE REVIEW

**Review by mattseddon — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -1,10 +1,11 @@
-.. image:: docs/assets/datachain-sys-theme.svg
-   :height: 48
-   :alt: DataChain logo
-
+================
```
Comment (mattseddon) [inline README.rst:1]: top line looks dodgy on light mode

image

Comment (mattseddon) [reply README.rst]: No, the line:

image

Comment (mattseddon) [reply README.rst]: I think it was fine before, I would just remove the extra top line

**Review by yathomasi — APPROVED** [review]
Looks great. Thanks!

**Pull request merged**